user_guides/jakartaee/src/main/jbake/content/intro.adoc

```diff
@@ -140,77 +140,122 @@ harness.
+* Jakarta Enterprise Beans (also, see optional below)
+* Jakarta Enterprise Web Services
+* Jakarta Expression Language
+* Jakarta Server Faces
+* Jakarta Interceptors
+* Jakarta JSON Binding
+* Jakarta JSON Processing
+* Jakarta Mail
+* Jakarta Messaging
+* Jakarta Persistence
+* Jakarta RESTful Web Services
+* Jakarta Security
+* Jakarta Server Pages
+* Jakarta Servlet
+* Jakarta Standard Tag Library
+* Jakarta Transactions
+* Jakarta WebSocket
+
+The following optional technologies may also tested via the Jakarta EE Platform TCK:
+
+* Jakarta Enterprise Beans and earlier entity beans and associated Jakarta Enterprise Beans QL 
+* Jakarta Enterprise Beans 2.x API group 
+* Jakarta Enterprise Beans Container Managed Persistence, Bean Managed Persistence
+* Jakarta Enterprise Web Services 
+* Jakarta SOAP with Attachments 
+* Jakarta XML Binding 
+* Jakarta XML Web Services
+
+Jakarta EE 10 Platform TCK provides certification verification of implementations contained in the Platform for the following component specifications:
+
+* Jakarta Annotations
+* Jakarta Authorization
+* Jakarta Connectors
+* Jakarta Enterprise Beans (including optional elements)
+* Jakarta Expression Language
+* Jakarta Interceptors
+* Jakarta Messaging
+* Jakarta Persistence
+* Jakarta Server Pages
+* Jakarta Servlet
+* Jakarta SOAP with Attachments
+* Jakarta Standard Tag Library
+* Jakarta Transactions
+* Jakarta Web Socket
+* Jakarta XML Web Services
 
 [[BHCGFHDI]][[jakarta-ee-10-web-profile-technologies-tested-with-jakarta-ee-10-platform-tck]]
 
 1.2.2 Jakarta EE 10 Web Profile Technologies Tested With Jakarta EE 10 Platform TCK
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The Jakarta EE 10 Platform TCK test suite can also be used to test compatibility for
-the following required Jakarta EE 10 Web Profile technologies:
-
-* Jakarta Authentication 3.0, Servlet Container Profile
-* Jakarta Bean Validation 3.0
-* Jakarta Common Annotations 2.1
-* Jakarta Contexts and Dependency Injection 4.0
-* Jakarta Concurrency 3.0
-* Jakarta Debugging Support for Other Languages 2.0
-* Jakarta Dependency Injection 2.0
-* Jakarta Enterprise Beans Lite 4.0, Lite
-* Jakarta Expression Language 5.0
-* Jakarta Faces 4.0
-* Jakarta JSON Processing 2.1
-* Jakarta JSON Binding 3.0
-* Jakarta Interceptors 2.1
-* Jakarta Persistence 3.1
-* Jakarta RESTful Web Services 3.1
-* Jakarta Security 3.0
-* Jakarta Server Pages 3.1
-* Jakarta Servlet 6.0
-* Jakarta Standard Tag Library 3.0
-* Jakarta Transactions 2.0
-* Jakarta WebSocket 2.1
+The Jakarta EE 10 Web Profile Specification defines the required component specifications.
+The full list with specification version requirements is defined in the Web Profile specification document (https://jakarta.ee/specifications/webprofile/10/), see heading 'Web Profile Definition', sub-heading 'Required Components'.
+
+The Jakarta EE 10 Platform TCK test suite provides partial compatibility verification for
```

Comment:
Again the following list is the complete list of web technologies. The Jakarta EE 10 Platform TCK does not test all of them.

Reply:
The Platform TCK may use any of the technologies. I think we should list them all. They may not be certified by running the TCK, but the TCK may use them in the associated tests.